psycop/common/feature_generation/loaders/raw/utils.py

```diff
@@ -90,6 +90,7 @@ def load_from_codes(
     shak_code: int | None = None,
     keep_code_col: bool = False,
     shak_sql_operator: str | None = None,
+    unique_count: bool = False,
```

Comment:
Jeg ville nok kalde den noget mere generelt, da man godt kunne forestille sig denne funktion også kunne bruges til andre ting end til at resolve unique_count aggregation metoden.

```suggestion
    add_code_to_output_col: bool = False,
```